src/seml/__main__.py

```diff
@@ -748,6 +749,46 @@ def description_list_command(ctx: typer.Context, update_status: UpdateStatusAnno
     collection_list_descriptions(ctx.obj['collection'], update_status=update_status)
 
 
+@app.command("export")
+@restrict_collection()
+def export_command(
+    ctx: typer.Context,
+    path: Annotated[
+        str,
+        typer.Option(
+            '-f',
+            "--file",
+            help="Specify path to export the collection to.",
+            is_flag=False,
+        ),
+    ] = None
+):
+    """
+        Export collection to JSON.
+    """
+    export_collection(ctx.obj['collection'], path)
+
+
+@app.command("import")
+@restrict_collection()
+def export_command(
+    ctx: typer.Context,
+    path: Annotated[
+        str,
+        typer.Argument(
+            help="Specify path to a JSON file to import a collection from.",
+            exists=True,
+            file_okay=True,
+            dir_okay=False,
+        ),
+    ]
+):
+    """
+        Import new collection from JSON.
```

Comment:
JSON/BSON, also allows adding to an existing collection, right?

Reply:
see discussion below